src/language_negotiation_lambda.js

```diff
@@ -36,20 +35,26 @@ exports.handler = (event, context, callback) => {
     }
 
     try {
-        if (headers[acceptLanguage]) {
-            var selectableLanguages = getSelectableLangugaes(headers[acceptLanguage].split(','));
-            var selectedLanguage = defaultLanguage;
+        headers[customHeaderName] = [defaultLanguage];
+    } catch(err) {
+         console.log("Error assigning default language: " + err);
+    }
```

Comment:
Looks like the only time this would be an error is if `headers` is never assigned. If `headers` is not assigned the rest of the code would also fail.